### ISSUE TITLE
Updated rule: "role attribute has valid value"

### ISSUE THREAD

**Issue by Jym77** [w3c/wcag-act#487]
The ACT-Rules community group has completed work on rule "`role` attribute has valid value". We would like to see this rule be added to the WCAG ruleset. You can find the rule here:

**Rule title**: rule: `role` attribute has valid value
**Rule id**: 674b10
**Link to the rule**: https://act-rules.github.io/rules/674b10
**License**: W3C Software And Document Notice And License
**Implementors**:
- axe-core
- QualWeb
- SortSite

If you'd like to contact me, please e-mail me at jym@siteimprove.com. (optional: You can also reach me on the W3C Community Slack, my name there is Jean-Yves Moyen)

**Comment by maryjom** [comment]
Opened [survey](https://www.w3.org/2002/09/wbs/93339/ACTRoleAttribute/results) due 15 Oct.

**Comment by maryjom** [comment]
Survey completed and discussed at the following ACT TF meetings:

- [1 October](https://www.w3.org/2020/10/01-wcag-act-minutes.html#item03)
- [8 October](https://www.w3.org/2020/10/08-wcag-act-minutes.html#item03)
- [15 October](https://www.w3.org/2020/10/15-wcag-act-minutes.html#item04)
-
